### PR TITLE
Reorganize and add new audit views

### PR DESCRIPTION
**Pull request by noahmoss**
* Reorganizes audit views under `/resources/migrations` with the following structure:

```
├── 000_migrations.yaml
└── instance_analytics_views
    ├── alerts
    │   └── v1
    │       └── alerts.sql
    ├── audit_log
    │   └── v1
    │       ├── audit_log.sql
    │       └── mysql-audit_log.sql
    ├── content
    │   └── v1
    │       ├── h2-content.sql
    │       ├── mysql-content.sql
    │       └── postgres-content.sql
    ├── dashboardcard
    │   └── v1
    │       └── dashboardcard.sql
    ├── databases
    │   └── v1
    │       └── databases.sql
    ├── fields
    │   └── v1
    │       └── fields.sql
    ├── group_members
    │   └── v1
    │       └── group_members.sql
    ├── query_log
    │   └── v1
    │       └── query_log.sql
    ├── subscriptions
    │   └── v1
    │       └── subscriptions.sql
    ├── tables
    │   └── v1
    │       └── tables.sql
    ├── users
    │   └── v1
    │       └── users.sql
    └── view_log
        └── v1
            └── view_log.sql
```

* Updates existing view migrations to read straight from the SQL files, so that there is a single source of truth
* Adds new views from https://github.com/metabase/metabase/pull/35251/files (and changes to existing views). We can treat these as the authoritative v1 versions of these views. Any changes should be made by copying the SQL files into `v2` directories.
* Small updates to the `lint-migrations-file` script to accommodate the new use of `:sqlFile` change type

### CODE REVIEW

File: resources/migrations/000_migrations.yaml [hunk 1]
```diff
@@ -16159,84 +15716,109 @@ databaseChangeLog:
...
-                      left join pulse_card on pulse.id = pulse_card.pulse_id
-                      left join pulse_channel on pulse.id = pulse_channel.pulse_id
-                  where pulse_card.dashboard_card_id is null
+        - sqlFile:
+            path: instance_analytics_views/subscriptions/v1/subscriptions.sql
+            relativeToChangelogFile: true
       rollback:
         - sql:
-            sql: drop view if exists v_alerts_subscriptions;
+            sql: drop view if exists v_subscriptoins;
```
Comment (luizarakaki) [inline resources/migrations/000_migrations.yaml:15736]: there is a typo here

File: resources/migrations/000_migrations.yaml [hunk 2]
```diff
@@ -16159,84 +15716,109 @@ databaseChangeLog:
...
+            relativeToChangelogFile: true
       rollback:
         - sql:
             sql: drop view if exists v_group_members;
 
   - changeSet:
       id: v48.00-032
       author: noahmoss
-      comment: Added 0.48.0 - new view v_alerts_subscriptions
+      comment: Added 0.48.0 - new view v_subscriptoins
```
Comment (luizarakaki) [inline resources/migrations/000_migrations.yaml:15729]: here too

File: bin/lint-migrations-file/src/change_set/strict.clj [hunk 3]
```diff
@@ -22,10 +22,17 @@
    ;; different DBMSes
    :sql-changes-for-different-
    (s/and
-    (s/+ :change.strict/dbms-qualified-sql-change)
+    (s/+ (s/alt :sql-change :change.strict/dbms-qualified-sql-change
+                :sqlFile-change :change.strict/dbms-qualified-sqlFile-change))
```
Comment (noahmoss) [inline bin/lint-migrations-file/src/change_set/strict.clj:26]: The changes to the spec in these files are just to ensure that the `lint-migrations-file` allows `sqlFile` changes to be split between different DB types, just like they can for `sql` changes

File: resources/liquibase.yaml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 databaseChangeLog:
-  - includeAll:
-      path: "migrations/"
+  - include:
+      file: "migrations/000_migrations.yaml"
```
Comment (noahmoss) [inline resources/liquibase.yaml:3]: `includeAll` resulted in all the SQL files in `resources/` to be run automatically by liquibase regardless of the DB type. I've changed it to just include the `000_migrations.yaml` file automatically, and the migrations file will pull in the rest of the SQL via `sqlFile` change types.

File: resources/migrations/instance_analytics_views/mysql-indexes.sql [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+create index idx_user_full_name
+    on core_user((concat(first_name, ' ', last_name)));
+
+-- view_log
+create index idx_view_log_timestamp
+    on view_log(timestamp);
+
+create index idx_view_log_entity_qualified_id
+    on view_log((concat(model, '_', model_id)));
```
Comment (escherize) [inline resources/migrations/instance_analytics_views/mysql-indexes.sql:37]: not sure if necessary, but this file and `resources/migrations/instance_analytics_views/postgres-indexes.sql` are missing a trailing newline

**Review by escherize — COMMENTED** [review]
LGTM

**Review by escherize — APPROVED** [review]
Meant to approve :)

**Pull request merged**